visual_behavior/data_access/loading.py

```diff
@@ -665,7 +796,7 @@ def get_cell_specimen_id_for_cell_roi_id(self, cell_roi_id):
 
 
 def get_ophys_dataset(ophys_experiment_id, include_invalid_rois=False, load_from_lims=False, load_from_nwb=True,
-                      get_extended_stimulus_presentations=True, get_behavior_movie_timestamps=False):
+                      get_extended_stimulus_presentations=False, get_behavior_movie_timestamps=False):
```

Comment:
Why change the default behavior? This will require updates to GLM code

Reply:
ill change it back. i just modified it because it makes loading the dataset slow. but if it affects GLM code, its better to have it stay as True.